<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaNoticias</class>
 <widget class="QWidget" name="VentanaNoticias">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>333</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>333</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Noticias</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>333</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>333</width>
     <height>51</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(255, 0, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0,0%);
font: 20pt &quot;Tw Cen MT Condensed&quot;;
border: None;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Noticias&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>182</x>
      <y>10</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/imagenes/imagenes/logo-udemedellin-footer.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../imagenes/logo-udemedellin-footer.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>61</y>
     <width>333</width>
     <height>439</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>333</width>
     <height>439</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>333</width>
     <height>439</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>333</width>
      <height>3</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>333</width>
      <height>0</height>
     </size>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="tbutton_noticias">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>370</y>
      <width>48</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>48</width>
      <height>33</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);
image: url(:/imagenes/imagenes/form.png);
border-radius: 20px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../imagenes/form.png</normaloff>../imagenes/form.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="tbutton_usuario">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>400</y>
      <width>48</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>48</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>48</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0%);
image: url(:/imagenes/imagenes/user.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../imagenes/user.png</normaloff>../imagenes/user.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="tbutton_reservas">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>400</y>
      <width>48</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>48</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>48</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0%);
image: url(:/imagenes/imagenes/calendar-clock.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../imagenes/calendar-clock.png</normaloff>../imagenes/calendar-clock.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="tbutton_opinion">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>48</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>48</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>48</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0%);
image: url(:/imagenes/imagenes/envelope.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../imagenes/envelope.png</normaloff>../imagenes/envelope.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <zorder>frame_2</zorder>
  <zorder>frame</zorder>
 </widget>
 <resources>
  <include location="imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
